src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/backfill/GetAlertsForCaseNotes.kt

```diff
@@ -35,11 +37,16 @@ class GetAlertsForCaseNotes(private val alertRepository: AlertRepository) {
 
   private fun Alert.forCaseNotes(): CaseNoteAlert {
     val (type, subtype) = typeInfo()
+    val activeFromChanged = auditEvents().lastOrNull { it.activeFromUpdated == true }
```

Comment:
If someone changed active from twice from NOMIS, this would only pick up the latter I think. Should this try and find the right one based on the actionedAt value initially?